src/satosa/micro_services/ldap_attribute_store.py

```diff
@@ -66,15 +65,17 @@ def __init__(self, config, *args, **kwargs):
 
         if "default" in config and "" in config:
             msg = """Use either 'default' or "" in config but not both"""
-            satosa_logging(logger, logging.ERROR, msg, None)
+            logline = lu.LOG_FMT.format(id=lu.get_session_id(None), message=msg)
```

Comment:
Why do we do `lu.get_session_id(None)`?
I think in all other cases, when `state` was `None` we skipped setting the `id` and just logged the `message`. Is that correct?

Shouldn't this be
```diff
-            satosa_logging(logger, logging.ERROR, msg, None)
+            logger.error(msg)
```

Reply:
Yes, that's correct. Let me change that.